packages/docs/src/routes/docs/cookbook/re-exporting-loaders/index.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Cookbook | Re-exporting loaders
+contributors:
+  - gioboa
+updated_at: '2023-12-15T11:00:00Z'
+created_at: '2023-12-15T11:00:00Z'
+---
+
+# Re-exporting loaders
+
+`routeAction$` and `routeLoader$` are typically declared in route boundary files such as layout.tsx, index.tsx and plugin.tsx inside the routesDir directory
```

Comment:
I think there are two examples:

1. My `indext.tsx` file is too pig and I want to pull code in a separate file as shown in this example.
2. I am writing third-party lib and just want to add an existing component. Now i need to re-export the component router functions. 

I don't think the second case is covered.

Reply:
✅ done